staging/07-querying-flink-tables/src/main/java/marketplace/CustomerService.java

```diff
@@ -0,0 +1,24 @@
+package marketplace;
+
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.TableResult;
+
+import static org.apache.flink.table.api.Expressions.$;
+
+public class CustomerService {
+    private final TableEnvironment env;
+
+    public CustomerService(TableEnvironment env) {
+        this.env = env;
+    }
+
+    public TableResult allCustomers() {
```

Comment:
I needed a way to make these methods at least somewhat "testable". Most of the examples of the Table API consist of nothing but a "main" method which isn't testable.

I opted to create these methods inside of Services so that I could write tests that target individual Flink queries/jobs.

However, if there is a better/more common way to organize a Flink Table API application, please let me know what it is and I can see if I can adapt.